src/components/Dropdown/index.tsx

```diff
@@ -75,7 +77,7 @@ export default function ({
       <span className={textStyles(colorfulCurrentValue)}>
         {staticPlaceholder || currentValue}
       </span>
-      {typeof staticPlaceholder === 'string' && (
+      {withArrow && (
```

Comment:
Now `SealVerse` is wrapped into `div displayFrom('xs')`, so we can't use type check